network/p2p/client.go

```diff
@@ -94,7 +94,7 @@ func (c *Client) AppRequest(
 		}
 
 		c.router.pendingAppRequests[requestID] = onResponse
-		c.router.requestID++
+		c.router.requestID += 2
```

Comment:
I think this should be configurable. I don't want this, for example, in the HyperSDK.

Reply:
I think we would plan on eventually reverting this once we had our VMs migrated. It feels like this would be a pretty strange configuration to expose, would you be fine with us just hard-coding this for now given that it's a temporary change?

Reply:
I guess the point is that the hypersdk doesn't _need_ this... but the hypersdk doesn't want this _not_ to exist. It's a noop for the hypersdk

Reply:
Patrick and I talked offline - going to make a separate PR to comment on `requestID` re-use.